.travis.yml

```diff
@@ -0,0 +1,37 @@
+language: go
+
+os:
+  - linux
+  - osx
+
+go:
+  - 1.8.3
+
+go_import_path: github.com/kubeapps/installer
+
+before_install:
+  - set -e
+
+script:
+  - make test
+  - make VERSION=${TRAVIS_TAG:-build-$TRAVIS_BUILD_ID} binary
+
+before_deploy:
+  - echo TARGET=$TARGET OS_NAME=$TRAVIS_OS_NAME GO_VERSION=$TRAVIS_GO_VERSION
+  - EXE_NAME=kubeapps-$(go env GOOS)-$(go env GOARCH)
```

Comment:
shouldn't you export this value?

Reply:
That's used for releasing cause I set EXE_NAME as the binary name for each platform (osx, linux)